gdsclient/tests/integration/conftest.py

```diff
@@ -17,6 +17,25 @@
     )
 
 
+def pytest_addoption(parser):
+    parser.addoption(
+        "--include-enterprise", action="store_true", help="include enterprise tests"
+    )
+
+
+@pytest.fixture
+def skip_if_not_enterprise(request) -> None:
+    if not request.config.getoption("--include-enterprise"):
+        pytest.skip()
```

Comment:
I wonder if we should name it `_skip_if_not_enterprise` to declare that it should be used directly. Or better yet, maybe we could define it inside of `enterprise()` below :)